easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,41 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+#
+# Fred Hutchenson Cancer Research Center Seattle WA
+easyblock = 'CMakeMake'
+
+name = 'OpenJPEG'
+version = '2.3.0'
+
+homepage = 'http://www.openjpeg.org/'
+description = """OpenJPEG is an open-source JPEG 2000 codec written in
+ C language. It has been developed in order to promote the use of JPEG 2000,
+ a still-image compression standard from the Joint Photographic Experts Group
+ (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as
+ a JPEG 2000 Reference Software."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/uclouvain/openjpeg/archive']
+sources = ['v%(version)s.zip']
+checksums = ['e0bd4ee9ab6e4f3a581fca5e01fbfb45a8ab9062baeddd1211741531b53fe07d']
+
+separate_build_dir = True
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('CMake', '3.11.4')
+]
+
+sanity_check_paths = {
+    'files': ['bin/opj_compress',
+              'bin/opj_decompress',
+              'bin/opj_dump',
+              'include/openjpeg-2.3/openjpeg.h',
```

Comment:
@fizwit Please avoid hardcoding (partial) version, fixed in https://github.com/fizwit/easybuild-easyconfigs/pull/4